solutions/07-querying-flink-tables/src/main/java/marketplace/Marketplace.java

```diff
@@ -0,0 +1,28 @@
+package marketplace;
+
+import io.confluent.flink.plugin.ConfluentSettings;
+import org.apache.flink.table.api.TableEnvironment;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+public class Marketplace {
+
+    public static void main(String[] args) throws Exception {
+        ConfluentSettings.Builder settings = ConfluentSettings.newBuilder("/cloud.properties");
+
+        TableEnvironment env = TableEnvironment.create(settings.build());
+        CustomerService customers = new CustomerService(env);
+        OrderService orders = new OrderService(env);
+
+        env.useCatalog("examples");
+        env.useDatabase("marketplace");
+
+        Arrays.stream(env.listTables()).forEach(System.out::println);
+
+        customers.allCustomers();
```

Comment:
When these queries are executed there is nothing to shut them down. For queries like this, with no sink, they will automatically shutdown a short time after the application terminates.

For complete streams that include a sink, they never shut down and will run forever.

Is that desirable? How would we recommend people manage these jobs to make sure they don't leak a bunch of queries and consume resources unnecessarily? How can we prevent someone from running the same job multiple times by accident and causing duplicate data in their topic? What is our recommended way of managing the lifecycle of these jobs?